bindings/pyroot/pythonizations/python/ROOT/pythonization/_roofit/_rooglobalfunc.py

```diff
@@ -117,6 +117,69 @@ def ZVar(*args, **kwargs):
     return RooFit._ZVar(*args, **kwargs)
 
 
+def Slice(*args, **kwargs):
+    # Redefinition of `Slice` for keyword arguments and converting python dict to std::map.
+    # The keywords must correspond to the CmdArg of the `Slice` function.
+    # The instances in the dict must correspond to the template argument in std::map of the `Slice` function.
+    from cppyy.gbl import RooFit
+
+    for i, arg_dict in enumerate(args):
+        if isinstance(arg_dict, dict):
+            args = list(args)
+            args[i] = _dict_to_std_map(arg_dict, "RooCategory*, std::string")
+
+    return RooFit._Slice(*args, **kwargs)
+
+
+def Import(*args, **kwargs):
+    # Redefinition of `Import` for keyword arguments and converting python dict to std::map.
+    # The keywords must correspond to the CmdArg of the `Import` function.
+    # The instances in the dict must correspond to the template argument in std::map of the `Import` function.
+    from cppyy.gbl import RooFit
+
+    def all_values_of_class(d, klass):
+        return all([isinstance(v, klass) for v in d.values()])
+
+    def _get_template_args(import_dict):
+        import ROOT
+
+        if all_values_of_class(import_dict, ROOT.RooDataSet):
+            value_class_name = "RooDataSet"
+        elif all_values_of_class(import_dict, ROOT.RooDataHist):
+            value_class_name = "RooDataHist"
+        elif all_values_of_class(import_dict, ROOT.TH1):
+            value_class_name = "TH1"
+        else:
+            raise TypeError(
+                """Unsupported value types in dictionary passed to RooFit.Import(import_dict). The imported objects need to be either: * all instances of RooDataSet * all instances of RooDataHist * all instances of TH1"""
```

Comment:
```suggestion
                """Unsupported value types in dictionary passed to RooFit.Import(import_dict). The imported objects need to be either:
  * all instances of RooDataSet
  * all instances of RooDataHist
  * all instances of TH1"""
```
I would suggest to make some line breaks in this string to make the bullet list clear.